modules/pel/peltool/peltool.py

```diff
@@ -307,6 +307,23 @@ def main():
 
         sys.exit(0)
 
+    if args.list:
+        if not os.path.isdir(args.list):
```

Comment:
If the user does not enter any directory location, then with current code it will take the default location. Which is that of BMC's PELs location. 
If we are not in BMC, or the entered directory by user doesn't exist, that could be a problem. Hence put a check before moving ahead.